frontend/packages/client/src/components/WalletConnectModal.js

```diff
@@ -5,13 +5,6 @@ import classnames from 'classnames';
 import sortBy from 'lodash/sortBy';
 import { ArrowRight, Close } from './Svg';
 
-const getWalletIcon = (provider) => {
-  if (provider?.name === 'Lilico') {
-    return 'https://raw.githubusercontent.com/Outblock/Lilico-Web/main/asset/logo-dis.png';
-  }
-  return `https://fcl-discovery.onflow.org${provider.icon}`;
-};
-
```

Comment:
I think we still need this function when `REACT_APP_FLOW_ENV=emulator`  because icons are not returning absolute paths on `provider.icon`